lib/blacklight/models/scorm_package.rb

```diff
@@ -0,0 +1,89 @@
+module Blacklight
+  class ScormPackage
+    attr_accessor(:entries, :manifest)
+
+    ##
+    # Scorm packages should include this string in the <schema> tag. We
+    # downcase, and remove spaces to be more consistant
+    ##
+    SCORM_SCHEMA = "adlscorm".freeze
```

Comment:
It is part of the scorm spec that this is included in the manifest. I thought it belonged as a class constant instead of just being a string inside of the method.